### PR TITLE
Update release notes for 13.2.0

### PR DESCRIPTION
**Pull request by PrajaktaPurohit**
Signed-off-by: Prajakta Purohit <prajakta@chef.io>

- [ ] New functionality includes tests
- [ ] All buildkite tests pass
- [ ] Full omnibus build and tests in buildkite pass
- [x] All commits have been signed-off for the Developer Certificate of Origin. See <https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco>
- [ ] PR title is a worthy inclusion in the CHANGELOG

### CODE REVIEW

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -8,6 +8,50 @@ This document contains release notes for the current major release and all patch
 For prior releases,
 see [PRIOR\_RELEASE\_NOTES.md](PRIOR_RELEASE_NOTES.md).
 
+## 13.2.0 (2020-4-14)
+
+###Improvements:
+- Azure supported for external postgres
+    In the previous release we added support for ssl while connecting to postgres.
```
Comment (tas50) [inline RELEASE_NOTES.md:15]:
```suggestion
    In the previous release we added support for ssl while connecting to PostgreSQL.
```

File: RELEASE_NOTES.md [hunk 2]
```diff
@@ -8,6 +8,50 @@ This document contains release notes for the current major release and all patch
 For prior releases,
 see [PRIOR\_RELEASE\_NOTES.md](PRIOR_RELEASE_NOTES.md).
 
+## 13.2.0 (2020-4-14)
+
+###Improvements:
```
Comment (btm) [inline RELEASE_NOTES.md:13]:
```suggestion
### Improvements:
```

File: RELEASE_NOTES.md [hunk 3]
```diff
@@ -8,6 +8,50 @@ This document contains release notes for the current major release and all patch
...
+- Azure supported for external postgres
```
Comment (btm) [inline RELEASE_NOTES.md:14]:
```suggestion
- Azure support for external PostgreSQL
```

File: RELEASE_NOTES.md [hunk 4]
```diff
@@ -8,6 +8,50 @@ This document contains release notes for the current major release and all patch
...
+- fix habitat db config for external database
+  (Thank you https://github.com/uwej711 for your contribution!)
+- Elasticsearch recipes do not create indexes at compile time
+- Change to log all errors that might happen with regarde to
+  elasticsearch reindex.
+
+### Updates:
+- Chef Infra Client: 15.5.17 -> 15.8.23
+- rack(oc-chef-pedant): 2.0.7 -> 2.0.8
+- Ruby: 1.6.11 -> 1.6.12
```
Comment (tas50) [inline RELEASE_NOTES.md:50]: This seems wrong

File: RELEASE_NOTES.md [hunk 5]
```diff
@@ -8,6 +8,50 @@ This document contains release notes for the current major release and all patch
...
+- rubyzip(oc-id): 1.2.3 -> 1.3.0 (fixes CVE-2019-16892)
+- terraform: 0.11 -> 0.12
```
Comment (tas50) [inline RELEASE_NOTES.md:52]: Is this just for the test infra?

File: RELEASE_NOTES.md [hunk 6]
```diff
@@ -8,6 +8,50 @@ This document contains release notes for the current major release and all patch
...
+    chef-server with different configurations and topologies.
+- External chef-server supports es version 6
+    Chef-Infra-Server could add indexes only for major versions 2 and 5
+    of elasticsearch.
+    With this changes we have added support to create indexes for elasticsearch
+    6 as well.
+- Cookstyle changes applied to the cookbooks
+
+### Bug Fixes:
+- Fix the FIPS regression
```
Comment (tas50) [inline RELEASE_NOTES.md:39]:
```suggestion
- Fix a regression that broke FIPS support in Chef Infra Server 13
```

File: RELEASE_NOTES.md [hunk 7]
```diff
@@ -8,6 +8,50 @@ This document contains release notes for the current major release and all patch
 For prior releases,
 see [PRIOR\_RELEASE\_NOTES.md](PRIOR_RELEASE_NOTES.md).
 
+## 13.2.0 (2020-4-14)
+
+### Improvements:
+- Azure support for external PostgreSQL
+    In the previous release we added support for ssl while connecting to PostgreSQL.
+    With this release we add the ability to be able to connect to external postgres
+    database in azure. We would like to thank https://github.com/zanecodes for
```
Comment (tas50) [inline RELEASE_NOTES.md:17]:
```suggestion
    database in Azure. We would like to thank https://github.com/zanecodes for
```

File: RELEASE_NOTES.md [hunk 8]
```diff
@@ -8,6 +8,50 @@ This document contains release notes for the current major release and all patch
 For prior releases,
 see [PRIOR\_RELEASE\_NOTES.md](PRIOR_RELEASE_NOTES.md).
 
+## 13.2.0 (2020-4-14)
+
+### Improvements:
+- Azure support for external PostgreSQL
+    In the previous release we added support for ssl while connecting to PostgreSQL.
+    With this release we add the ability to be able to connect to external postgres
```
Comment (tas50) [inline RELEASE_NOTES.md:16]:
```suggestion
    With this release we add the ability to be able to connect to external PostgreSQL
```

File: RELEASE_NOTES.md [hunk 9]
```diff
@@ -8,6 +8,50 @@ This document contains release notes for the current major release and all patch
...
+     * Set on-marked-down shutdown-session to avoid stale sessions to
+       previous leaders living longer than they need to.
+- Integration testing pipeline
+    We have put a lot of effort into creating a test pipeline with the
+    test infrastructure previously created. This runs multiple scenarios for
+    chef-server with different configurations and topologies.
+- Chef-Infra-Server supports Elasticsearch version 6 for external Elasticsearch
+    Chef-Infra-Server could add indexes only for major versions 2 and 5
+    of elasticsearch.
+    With this changes we have added support to create indexes for elasticsearch
```
Comment (tas50) [inline RELEASE_NOTES.md:34]:
```suggestion
    With this changes we have added support to create indexes for Elasticsearch
```

File: RELEASE_NOTES.md (hunk 4 above)
Comment (PrajaktaPurohit) [reply RELEASE_NOTES.md]: updated -that is the ruby for oc-id

File: RELEASE_NOTES.md (hunk 5 above)
Comment (PrajaktaPurohit) [reply RELEASE_NOTES.md:52]: yes.

Comment (PrajaktaPurohit) [reply RELEASE_NOTES.md:52]: deleted.

File: RELEASE_NOTES.md [hunk 10]
```diff
@@ -8,6 +8,50 @@ This document contains release notes for the current major release and all patch
...
+     * Set connect, client, server, and tunnel timeouts to reasonable defaults.
+     * Set client-fin and server-fin to try to mitigate connection pile-ups
+       in the case of failing frontend services.
+     * Set on-marked-down shutdown-session to avoid stale sessions to
+       previous leaders living longer than they need to.
+- Integration testing pipeline
+    We have put a lot of effort into creating a test pipeline with the
+    test infrastructure previously created. This runs multiple scenarios for
+    chef-server with different configurations and topologies.
+- Chef-Infra-Server supports Elasticsearch version 6 for external Elasticsearch
```
Comment (tas50) [inline RELEASE_NOTES.md:32]:
```suggestion
- Chef Infra Server supports Elasticsearch version 6 for external Elasticsearch
```

File: RELEASE_NOTES.md [hunk 11]
```diff
@@ -8,6 +8,50 @@ This document contains release notes for the current major release and all patch
...
+    more responsive. The changes include:
+     * Set connect, client, server, and tunnel timeouts to reasonable defaults.
+     * Set client-fin and server-fin to try to mitigate connection pile-ups
+       in the case of failing frontend services.
+     * Set on-marked-down shutdown-session to avoid stale sessions to
+       previous leaders living longer than they need to.
+- Integration testing pipeline
+    We have put a lot of effort into creating a test pipeline with the
+    test infrastructure previously created. This runs multiple scenarios for
+    chef-server with different configurations and topologies.
```
Comment (tas50) [inline RELEASE_NOTES.md:31]: Chef Infra Server

File: RELEASE_NOTES.md [hunk 12]
```diff
@@ -8,6 +8,50 @@ This document contains release notes for the current major release and all patch
...
+- Chef Infra Server supports Elasticsearch version 6 for external Elasticsearch
+    Chef-Infra-Server could add indexes only for major versions 2 and 5
```
Comment (tas50) [inline RELEASE_NOTES.md:33]: Chef Infra Server

File: RELEASE_NOTES.md [hunk 13]
```diff
@@ -8,6 +8,50 @@ This document contains release notes for the current major release and all patch
...
+       previous leaders living longer than they need to.
+- Integration testing pipeline
+    We have put a lot of effort into creating a test pipeline with the
+    test infrastructure previously created. This runs multiple scenarios for
+    Chef Infra Server with different configurations and topologies.
+- Chef Infra Server supports Elasticsearch version 6 for external Elasticsearch
+    Chef Infra Server could add indexes only for major versions 2 and 5
+    of Elasticsearch.
+    With this changes we have added support to create indexes for Elasticsearch
+    6 as well.
```
Comment (markan) [inline RELEASE_NOTES.md:36]:
```suggestion
Chef Infra Server previously supported index creation for ElasticSearch versions 2 and 5. We now support index creation for ElasticSearch 6 as well.
```

**Pull request merged**